ladybug_charts/to_figure.py

```diff
@@ -0,0 +1,74 @@
+"""Create plotly figures from pandas Dataframe."""
+
+import numpy as np
+import plotly.io as pio
+import plotly.graph_objects as go
+from plotly.graph_objects import Figure
+from ._to_dataframe import heatmap_dataframe
+from math import ceil, floor
+from ladybug.datacollection import HourlyContinuousCollection
+from ladybug_pandas.series import Series
+
+
+def heatmap(hourly_data: HourlyContinuousCollection,
+            min_range: float = None, max_range: float = None) -> Figure:
```

Comment:
Should we use legend parameters here directly so we can also pass other values?

Also what happens with conditional statements in this case? Am I right that in this iteration we cannot support those because we only support the continuous version of Data Collection?

Reply:
I thought I will think about the legend parameters later. Currently, we're only changing the data range.

I added support for HourlyDiscontinuousCollection to Heatmap [here](https://github.com/ladybug-tools/ladybug-charts/pull/16)